### DISCUSSION TITLE
[Question] (When) is there an update planned to address the library vulnerabilities

### DISCUSSION THREAD

**Discussion by diderich** [YetiForceCompany/YetiForceCRM#13996] [General]
```
System warnings 
Vulnerabilities in libraries
Vulnerabilities in libraries were detected, it is recommended to update them quickly. 
Vulnerabilities:
phpoffice/phpspreadsheet(1.15.0):
XSS Vulnerability in HTML Writer CVE-2020-7776
```

**Comment by mariuszkrzaczkowski** [comment]
This error is not present in CRM

**Reply by simple-AMS** [reply]
> This error is not present in CRM

I can assure you it is @mariuszkrzaczkowski and I even sent an email to you guys about it. @diderich is correct. See screenshot:

![security_issue](https://user-images.githubusercontent.com/42642523/103654460-81838400-4f2b-11eb-85fb-3987fb0e5a80.png)

**Reply by mariuszkrzaczkowski** [reply]
my point is that the crm system is immune to this vulnerability
we don't use the vulnerable part of the library

**Reply by simple-AMS** [reply]
> my point is that the crm system is immune to this vulnerability
> we don't use the vulnerable part of the library

@mariuszkrzaczkowski That may indeed be the case. However, once ANYONE sees an alert regarding any type of vulnerability no matter how small, how safe and secure do you think they really feel regarding the use of a product?

**Reply by mariuszkrzaczkowski** [reply]
> @mariuszkrzaczkowski That may indeed be the case. However, once ANYONE sees an alert regarding any type of vulnerability no matter how small, how safe and secure do you think they really feel regarding the use of a product?

@acsinsurancegroup
Do you prefer to know about the potential danger or live blissfully unaware, like it is in many CRM/ERP systems?

Do you know how many threats are there to harm servers? Sometimes one outdated app in the OS is enough. You can really do a lot at the PHP level.

We build the system in such a way that it informs about all dangers and errors, so that people can act quickly. Not enough people follow our GitHub to know what's happening and how to react.

**Comment by diderich** [comment]
It is event in the stable version (corrected on gitdeveloper.yetiforce.com):

![vulnerability](https://user-images.githubusercontent.com/10230144/104034377-18bd3580-51d1-11eb-9d7d-7401ecf54b88.png)

**Comment by jmginfor** [comment]
Same here for the issue #13996


![2021-01-21 18_30_56-Age of Empires II_ Definitive Edition](https://user-images.githubusercontent.com/66257334/105425324-26ce7580-5c17-11eb-8813-65600ebf65b5.png)


![2021-01-21 18_31_28-Age of Empires II_ Definitive Edition](https://user-images.githubusercontent.com/66257334/105425323-2635df00-5c17-11eb-9055-d89cfac0654d.png)

**Comment by simple-AMS** [comment]
@diderich @jmginfor I guess there's no plans to address it at all?

**Reply by diderich** [reply]
Unfortunately, the vulnerability check no longer works:

![image](https://user-images.githubusercontent.com/10230144/106644750-c979e300-658b-11eb-9097-7b4c695026d7.jpeg)

![image](https://user-images.githubusercontent.com/10230144/106644764-ced72d80-658b-11eb-91b4-912dc5305b36.jpeg)

**Reply by rskrzypczak** [reply]
some people might say the problem solved itself :)
..but seriously..  Work is underway on a new alternative.

>(When) is there an update planned to address the library vulnerabilities

With version 6.2, I suppose

**Comment by simple-AMS** [comment]
@diderich So outta sight, outta mind is the approach they took. That's great!
